### PR TITLE
TSK-1986: Enable automated cleaning of disks

### PR DESCRIPTION
**Pull request by dougszumski**
Each time an Ironic node is returned to the 'available' state
all disks will be erased, using ATA secure erase where
available. No changes to the Ironic Python Agent ramdisk are
required for this to work. The cleaning steps are listed on
the nodes driver_info.

### CODE REVIEW

**Comment by dougszumski** [comment]
For reference the logic this enables is here:

https://github.com/openstack/ironic-python-agent/blob/stable/pike/ironic_python_agent/hardware.py#L750

File: etc/kayobe/kolla/config/ironic/ironic-conductor.conf [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Enable automated cleaning. To save time this uses ATA secure erase when
+# supported.
+
+[conductor]
+automated_clean=true
```
Comment (markgoddard) [inline etc/kayobe/kolla/config/ironic/ironic-conductor.conf]: Wax on, wax off. Scrub those nodes nice and clean.

File: etc/kayobe/kolla/config/ironic/ironic-conductor.conf [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Enable automated cleaning. To save time this uses ATA secure erase when
+# supported.
```
Comment (markgoddard) [inline etc/kayobe/kolla/config/ironic/ironic-conductor.conf:3]: nit: If we add any more options in here, this comment won't make much sense at the file level - move to above the option?

Comment (dougszumski) [reply etc/kayobe/kolla/config/ironic/ironic-conductor.conf:3]: Right you are sir. Fixed.

**Review by markgoddard — APPROVED** [review]

**Pull request merged**